indico/modules/rb_new/client/js/components/pages/Calendar.jsx

```diff
@@ -63,9 +63,8 @@ class Calendar extends React.Component {
         const {date, fetchCalendar, setDate} = this.props;
         if (!date) {
             setDate(moment());
-        } else {
-            fetchCalendar();
         }
+        fetchCalendar();
```

Comment:
Are you sure this won't dispatch the fetch twice? See my comment [down here](https://github.com/indico/indico/blob/cdfb1c9c12bee72103e07e3d332f4aa77c91edf7/indico/modules/rb_new/client/js/components/pages/Calendar.jsx#L120-L121)